### PR TITLE
Add hype emote

### PR DESCRIPTION
**Pull request by RDIL**
I did my best, plz don’t cyberbully me.  It’s basically the same as sk1ers yeet emote.  This is my first time doing something this big in java :)
If any changes need to be made feel free to push to my fork.

### CODE REVIEW

**Comment by RDIL** [comment]
no travis why fail :(

**Comment by simonri** [comment]
> 
> 
> no travis why fail :(

You forgot a semicolon in HyperiumHandlers.java at line 112
`private HypeHandler hypeHandler`
should be:
`private HypeHandler hypeHandler;`

**Comment by RDIL** [comment]
(Thanks @simonri for fixing the bug)

File: src/main/java/cc/hyperium/handlers/handlers/animation/HypeHandler.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            GlStateManager.scale(4, 4, 4);
+            worldrenderer.begin(7, DefaultVertexFormats.POSITION_COLOR);
+            worldrenderer.pos((double) (-j - 1), (double) (-1 + i), 0.0D).color(0.0F, 0.0F, 0.0F, 0.25F).endVertex();
+            worldrenderer.pos((double) (-j - 1), (double) (8 + i), 0.0D).color(0.0F, 0.0F, 0.0F, 0.25F).endVertex();
+            worldrenderer.pos((double) (j + 1), (double) (8 + i), 0.0D).color(0.0F, 0.0F, 0.0F, 0.25F).endVertex();
+            worldrenderer.pos((double) (j + 1), (double) (-1 + i), 0.0D).color(0.0F, 0.0F, 0.0F, 0.25F).endVertex();
+            tessellator.draw();
+            GlStateManager.enableTexture2D();
+
+            fontrenderer.drawString(string, -j, 0, Settings.REACH_COLOR_TYPE.equalsIgnoreCase("RGB") ? new Color(Settings.REACH_RED, Settings.REACH_GREEN, Settings.REACH_BLUE).getRGB() : Hyperium.INSTANCE.getModIntegration().getLevelhead().getRGBColor(), true);
```
[the remark this answers is not in the document]
Comment (RDIL) [reply src/main/java/cc/hyperium/handlers/handlers/animation/HypeHandler.java]: I don’t see anything wrong...

[the remark this answers is not in the document]
Comment (asbyth) [reply src/main/java/cc/hyperium/handlers/handlers/animation/HypeHandler.java]: reachdisplay stuff 🤔

File: src/main/java/cc/hyperium/gui/GuiDances.java [hunk 2]
```diff
@@ -63,6 +63,18 @@ public GuiDances() {
...
         });
+        
+        this.handlers.put("Hype", netty -> {
+             Hyperium.INSTANCE.getHandlers().getHypeHandler().hype(Minecraft.getMinecraft().thePlayer.getUniqueID());
+             NettyClient client = NettyClient.getClient();
+             if (client != null && netty) {
+                 client.write(ServerCrossDataPacket.build(new JsonHolder().put("type", "hype").put("hyping", true)));
+             }
+         });
+         this.cancel.put("Hype", () -> {
```
Comment (asbyth) [inline src/main/java/cc/hyperium/gui/GuiDances.java]: uhhh shouldnt this have a stopper like the floss does below

**Comment by RDIL** [comment]
Idk ask sk1er

File: src/main/java/cc/hyperium/gui/GuiDances.java (hunk 2 above)
Comment (Sk1er) [reply src/main/java/cc/hyperium/gui/GuiDances.java]: yeah it needs one

Comment (RDIL) [reply src/main/java/cc/hyperium/gui/GuiDances.java]: ok I’ll look into it

**Comment by RDIL** [comment]
should be fixed now!

**Comment by RDIL** [comment]
Hmm build is erroring due to failed forge jar download

**Comment by RDIL** [comment]
Could someone please help out idk how to fix it

**Pull request closed without merging**